src/Microsoft.VisualStudio.Composition/Reflection/ResolverExtensions.cs

```diff
@@ -14,6 +14,7 @@ namespace Microsoft.VisualStudio.Composition.Reflection
     public static class ResolverExtensions
     {
         private const BindingFlags AllInstanceMembers = BindingFlags.Instance | BindingFlags.Public | BindingFlags.NonPublic;
+        private const BindingFlags AllMembers = AllInstanceMembers | BindingFlags.Static;
```

Comment:
We always could when using fast reflection because the static flag didn't matter -- we got whatever method was for a given metadata token. Now that we're adding slow reflection paths, we have to specify what kind of method we're looking for, and since we supported static methods before, we need to support them with the slow API use.